src/components/TextField/TextField.vue

```diff
@@ -57,9 +57,9 @@ and `minlength`.
 			<Lock :size="16" />
 		</TextField>
 		<div class="external-label">
-			<label for="$refs.textField.id">External label</label>
+			<label for="textField">External label</label>
 			<TextField :value.sync="text5"
-				ref="textField"
+				id="textField"
```

Comment:
This needs to be unique by adding a random id. Otherwise we will have the same id multiple times on the page (in the likely case of multiple textfields), which is invalid.

Reply:
This in the docs/example part.
I had the same issue 😄

Reply:
Ah, right. Thanks for the hint.